src/Http/RequestFactory.php

```diff
@@ -238,6 +238,7 @@ public function createHttpRequest(): Request
 			} else {
 				if (!empty($_SERVER['HTTP_X_FORWARDED_PROTO'])) {
 					$url->setScheme(strcasecmp($_SERVER['HTTP_X_FORWARDED_PROTO'], 'https') === 0 ? 'https' : 'http');
+					$url->setPort($url->getScheme() === 'https' ? 443 : 80);
```

Comment:
Wow, it's great, I can remove theses lines.

https://github.com/componette/dockerfiles/commit/10a45113513af68f2ec9190cf456986906f73ff9